Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/Assembly.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly:InternalsVisibleTo("Frends.FixedWidthFlatFile.Parse.Tests")]
```

Comment:
What is the function of this file?

Reply:
The function of this file is to grant test assembly access to internal members of main assembly.